pkg/output/output.go

```diff
@@ -48,29 +48,42 @@ func (f *fsWriter) Writer(parentPath, name string, permissions os.FileMode) (Clo
 	}, writer, nil
 }
 
-// NewFSWriter returns an Outputter which writes the files and directories to a specified location.
-func NewFSWriter(path string) (Outputter, error) {
+func EnsureOutputPath(path string, deleteIfExists bool) error {
 	info, err := os.Stat(path)
 	if err != nil {
-		if !os.IsNotExist(err) {
-			return nil, err
+		if os.IsNotExist(err) {
+			return os.Mkdir(path, 0700)
+		}
+		return err
+	} else {
```

Comment:
You don't need to have the `else` branch anymore. Or rather whatever is in the else branch can be after the `if`. So the steps would be:
1. Stat the input path.
2. If error check if not exists and create if it does not. Either way return.
3. From stat info check if input is directory. if not return error. We don't want to accidentally delete something which is not an output directory.
4. check if directory is empty.
4.1 if not empty and delete flag is set delete
4.2 else return error
5. create directory
6. return

Reply:
done